### ISSUE TITLE
make regen broken: dial tcp 127.0.0.1:443: connect: connection refused

### ISSUE THREAD

**Issue by Jille** [kyleconroy/sqlc#2367]
### Version

Other

### What happened?

Running `make regen` seems to depend on a remote server for which the setup isn't documented. Can I simply keep doing `make regen` without remote somehow?

Strangely this does still seem to be working locally on my mac, but not on my linux server.

cc @andrewmbenton 

### Relevant log output

```shell
$ go run ./scripts/regenerate/ --no-remote
2023/06/25 22:25:46 internal/endtoend/testdata/remote_missing_semicolon/mysql: sqlc-dev generate failed
rpc error: connection error: desc = "transport: Error while dialing: dial tcp 127.0.0.1:443: connect: connection refused"
exit status 1
```

### What operating system are you using?

Linux

**Comment by Jille** [comment]
Nevermind me. I just had an old version and I can't git.

**Comment by andrewmbenton** [comment]
Yeah I don't have this issue on current main. There is a remote execution feature that I worked on, only documented in the 1.18.0 release changelog: https://docs.sqlc.dev/en/stable/reference/changelog.html#remote-code-generation

I had written a test for it that did depend on a locally-running service, but we removed the test at some point.
